packages/itwin/tree-widget/src/components/trees/stateless/categories-tree/CategoriesTree.tsx

```diff
@@ -3,6 +3,7 @@
  * See LICENSE.md in the project root for license terms and full copyright notice.
  *--------------------------------------------------------------------------------------------*/
 
+import "../Tree.scss";
```

Comment:
I think this should be imported in `CategoriesTreeComponent` and `ModelsTreeComponent`

Reply:
Fixed.